lib/utf8_sanitizer.rb

```diff
@@ -22,7 +24,23 @@ def call(env)
 
   private
 
-  def invalid_strings(values)
+  # @param [Hash] params
+  def params_have_null_byte?(params)
+    params.values.any? { |value| contains_null_byte?(value) }
+  end
+
+  def contains_null_byte?(param)
+    case param
+    when Hash
+      param.values.any? { |value| contains_null_byte?(value) }
```

Comment:
Could / should this be...

```suggestion
      params_have_null_byte?(param)
```

Reply:
I did the opposite, I removed `params_have_null_byte` in 94fa29697819d0c4b2d01770b4959cf7dfc79490

Reply:
> I did the opposite, I removed `params_have_null_byte` in [94fa296](https://github.com/18F/identity-idp/commit/94fa29697819d0c4b2d01770b4959cf7dfc79490)

I like that better 👍